### ISSUE TITLE
Reputation system [SUGGESTION]

### ISSUE THREAD

**Issue by cI0ak** [gitrous/origins#75]
**Quick Explanation**
Reputation system is an OOC system that is based on bad acts and good acts. It will affect different things as a civilian and can be both good and bad.

**Where to win reputation and where to lose reputation**
As told, this system is based on good and bad acts. So an occasion where you could win reputation is for example when you're working. Each shift you do could give you around 1 or 2 reputation points depending on where you're working. Another occasion where you could win reputation is by buying items from a store and the reputation points you get for it is based on the price. ((OBS. You can only earn points from buying items in a store each 15 minutes to prevent abuse.))

For this system to work we will also need to have occasions where you lose reputation points. One of these occasions could be when killing/damaging someone. Each kill could be around 5 reputation points so if you kill one person as in self defence it won't really hurt but if you're a terrorist mass killing it will hurt. This will also somewhat reduce rk. Hurting people should end up in a reputation point loss of 1 point per 25 damage. ((NOTE THAT IF THE PERSON YOU'RE KILLING IS AN OFFICIAL FACTION MEMBER YOU WILL NOT LOSE ANY REP POINTS)) Another occasion that would make you lose rep is when you leave your job. This should take about 15 reputation points. ((DOES NOT TAKE POINTS IF YOU'RE BEING FIRED))
Last occasion I'm gonna bring up here is vandalism. If you vandalise something (in this case shoot out windows or damage others cars) you will lose the right amount of reputation points for each thing. For example if you break a window you will lose 1 reputation point for each window. Next example is hurting another characters car, if you destroy the car fully you will lose 75 reputation points (blow it up) but otherwise it will only cost 1 reputation point per 2 damage dealen.

**What the system would look like**
When making your character you would start at a reputation of 1, meaning you can start working, but if you kill someone before working (which is highly unlikely) your life will be ruined and you'll need to beg for cash to buy food which will give you reputation and you can work and everything is fine. After working you've maybe gathered yourself around 25 reputation points meaning that you're on a pretty safe level to start doing other things than working.

About where you are on the charts is marked with an X.
                                               X
...[-100] - - - - [-50] - - - - [0] - - - - [50] - - - - [100]...

As you can see, you can travel up or down the charts depending on your actions.

**The advantages or disadvantages**
The factors that are affected by the reputation system are the following.
- Salary Percentage
- Employment
- Housing

Salary Percentage

Your reputation points will affect the percentage of your salary. 25% of the salary will be fought of with the reputation system, for example if you've got 50 reputation points you will get a 25% higher salary than a neutral reputation (0). And if you'd end up on -50 reputation you'd lose 25% of the salary that a neutral (0) gets. This is not based on stages, the reputation points you have will be calculated and will be given a percentage, just keep in mind that the max percentage + and - you can get is 25%.

Employment

The Newspaper boy work is the easiest to get into and the only rep required to get into it is 0, just because it's a starter job and everyone needs a beginning. And also as spoken about, there will be a banker job getting added and it's said to be the best civilian job. To work there I think instead of TIB you'd need around 500 reputation points. Now when it comes to the salary percentage there the neutral joining reputation is 500, and so the salary percentage should be based as if 500 is neutral. (550 reputation is needed for a 25% raise and 450 reputation is needed for a 25% lowering.) 
(OBS. You do not lose your job if you go lightly under the joining reputation, but if you get 150 reputation below you will be fired.) And as mentioned earlier, you will not lose any reputation from being fired, only by leaving the job.

Housing

The Zapodoslavia apartments are the cheapest apartments on the market and require atleast -25 or higher reputation points to live there. The apartment holders do not really care who lives there since the apartments themselves do not cost alot of money. But, if you'd like to live on finer places you first off need more cash, and second you need a higher reputation. The higher end the district is = the more reputation points you need. Each district should have about a 150 reputation change needed to live in their apartments. If you get 40 reputation points below the needed reputation you will be evicted.

Sorry that I had to make this suggestion long, if you think it would be a good addition please comment and if you think there could be something changed please comment that too.
Community & Game Moderator,
cI0ak.

**Comment by gitrous** [comment]
Other than Housing (the state rents out all properties), I like this. Obviously the exact numbers will need some tweaking if I do add this.

**Comment by cI0ak** [comment]
yeah the numbers here are more like examples, nothing counted on and nothing balanced
